viewer/lib/entities.js

```diff
@@ -13,6 +13,29 @@ function getEntityMesh (entity, scene) {
   } else if (entity.type) {
     try {
       const e = new Entity('1.16.4', entity.type, scene)
+
+      if ((entity.type === 'player')) {
```

Comment:
what about just checking if username is defined ?

Reply:
done